features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/components/event/TimelineItemImageView.kt

```diff
@@ -28,14 +28,17 @@ import io.element.android.features.messages.impl.timeline.model.event.TimelineIt
 import io.element.android.libraries.designsystem.preview.ElementPreviewDark
 import io.element.android.libraries.designsystem.preview.ElementPreviewLight
 import io.element.android.libraries.matrix.ui.media.MediaRequestData
+import kotlin.math.max
 
 @Composable
 fun TimelineItemImageView(
     content: TimelineItemImageContent,
     modifier: Modifier = Modifier,
 ) {
+    // TODO place this value somewhere else?
+    val minHeight = max(100, content.height ?: 0)
```

Comment:
Do we have somewhere where we add these kind of constants?

Reply:
Not for now, but we could have dimension param in the ElementTheme?